### PR TITLE
Update installers for jdk-17.0.3+7

### PR DESCRIPTION
**Pull request by Haroon-Khel**
Updated the jdk17 installers to use jdk-17.0.3+7

### CODE REVIEW

**Review by sxa — DISMISSED** [review]
Can you update that changelog too as in https://github.com/adoptium/installer/pull/425/files

File: linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 1]
```diff
@@ -1,9 +1,9 @@
-%global upstream_version 17.0.2+8
+%global upstream_version 17.0.3+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
-#  17.0.1.0.0___12 == 17.0.1.0.0+12
+#  17.0.1.0.0___12 == 17.0.3.0.0+7
 %global spec_version 17.0.2.0.0.8
```
Comment (jiekang) [inline linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec:7]: @Haroon-Khel should this spec_version be changed as well? I see it changed in the redhat spec above

Comment (Haroon-Khel) [reply linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec:7]: Youre right. I changed the wrong line by mistake

**Review by sophia-guo — APPROVED** [review]
LGTM

**Comment by smlambert** [comment]
We many want to create a new PR for this, since merging is blocked by @sxa and he is away now for a week.

File: linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec [hunk 2]
```diff
@@ -1,10 +1,10 @@
-%global upstream_version 17.0.2+8
+%global upstream_version 17.0.3+7
 # Only [A-Za-z0-9.] allowed in version:
 # https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_upstream_uses_invalid_characters_in_the_version
 # also not very intuitive:
 #  $ rpmdev-vercmp 17.0.1.0.1___17.0.1.0+12
-#  17.0.1.0.0___12 == 17.0.1.0.0+12
-%global spec_version 17.0.2.0.0.8
+#  17.0.1.0.0___12 == 17.0.3.0.0+7
```
Comment (karianna) [inline linuxNew/jdk/suse/src/main/packaging/temurin/17/temurin-17-jdk.spec:6]:
```suggestion
#  17.0.3.0.0___7 == 17.0.3.0.0+7
```

**Review by karianna — APPROVED** [review]
LGTM - should be able to merge now?

**Comment by smlambert** [comment]
@karianna - the problem is not with this PR, which is now ready to merge, it is that a reviewer has put a request for changes and left for vacation, which blocks this PR from being merged.  So I suggested a fresh PR to expedite the publishing of installers.

**Comment by karianna** [comment]
@smlambert I applied some admin power to make that review go away :-).  Feel free to merge.

**Pull request merged**